CI is failing

Seems that CI is failing because of updates in the `zenoh` crate.

```
error[E0432]: unresolved import `zenoh_util::sync`
Error:   --> zenoh-flow/src/runtime/dataflow/instance/runners/mod.rs:36:17
   |
36 | use zenoh_util::sync::Signal;
   |                 ^^^^ could not find `sync` in `zenoh_util`

For more information about this error, try `rustc --explain E0432`.
``` 

It should be easy as using `zenoh_sync::Signal`

--- a/zenoh-flow/Cargo.toml
+++ b/zenoh-flow/Cargo.toml
@@ -44,6 +44,7 @@ url = "2.2.2"
 uuid = { version = "0.8", features = ["serde", "v4"] }
 zenoh = { git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "master"}
 zenoh-flow-derive = {path = "../zenoh-flow-derive"}
+zenoh-sync = { git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "master" }
 zenoh-util = { git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "master" }
 znrpc-macros = { git = "https://github.com/atolab/zenoh-rpc.git", branch = "master" }
 zrpc = { git = "https://github.com/atolab/zenoh-rpc.git", branch = "master" }

--- a/zenoh-flow/src/runtime/dataflow/instance/runners/connector.rs
+++ b/zenoh-flow/src/runtime/dataflow/instance/runners/connector.rs
@@ -24,6 +24,7 @@ use crate::runtime::InstanceContext;
 use crate::{NodeId, PortId, PortType, ZFError, ZFResult};
 use async_trait::async_trait;
 use futures::prelude::*;
+use zenoh::publication::CongestionControl;
 
 #[derive(Clone)]
 pub struct ZenohSender {
@@ -74,6 +75,7 @@ impl ZenohSender {
                     .runtime
                     .session
                     .put(&self.record.resource, serialized)
+                    .congestion_control(CongestionControl::Block)
                     .await?;
             }
         } else {

--- a/zenoh-flow/src/runtime/dataflow/instance/runners/mod.rs
+++ b/zenoh-flow/src/runtime/dataflow/instance/runners/mod.rs
@@ -33,7 +33,7 @@ use std::collections::HashMap;
 use std::ops::Deref;
 use std::pin::Pin;
 use std::task::{Context, Poll};
-use zenoh_util::sync::Signal;
+use zenoh_sync::Signal;
 
 #[derive(Clone, Debug, PartialEq)]
 pub enum RunnerKind {

--- a/zenoh-flow/src/runtime/dataflow/instance/runners/source.rs
+++ b/zenoh-flow/src/runtime/dataflow/instance/runners/source.rs
@@ -30,6 +30,7 @@ use async_trait::async_trait;
 use std::collections::HashMap;
 use std::time::Duration;
 use uhlc::{Timestamp, NTP64};
+use zenoh::publication::CongestionControl;
 
 #[cfg(target_family = "unix")]
 use libloading::os::unix::Library;
@@ -142,6 +143,7 @@ impl SourceRunner {
             .runtime
             .session
             .put(&resource_name, serialized)
+            .congestion_control(CongestionControl::Block)
             .await?;
 
         Ok(())

